### ISSUE TITLE
Framework7 - Vue - Webpack Cordova Template

### ISSUE THREAD

**Issue by centrual** [framework7io/framework7-vue#44]
Hello Vladimir and Framework7 community;

I made cordova template for framework7 - vue - webpack combination. Please check at: https://github.com/centrual/cordova-template-framework7-vue-webpack

**Sample Installation:** `cordova create test_dir --template cordova-template-framework7-vue-webpack`

**Commands:** You can use any cordova commands. Examples:
``` bash
  cordova platform (add|rm) (android|ios|browser)
  cordova (run|build|prepare) (android|ios|browser) [-- --live-reload]
```

Have a great day!

**Comment by henninghorn** [comment]
Wow, this is amazing, @centrual! Great work!

**Comment by centrual** [comment]
Thank you @henninghorn 💛

**Comment by valnub** [comment]
Great work! It might be worth checking out Holly Schinsky's template as well: https://github.com/hollyschinsky/phonegap-template-framework7-vue

Also, I created a blog post about these new templates: https://www.timo-ernst.net/blog/2017/01/15/phonegapcordova-templates-for-vuejs-with-framework7/

**Comment by centrual** [comment]
I updating the template with webpack-dev-server and hot-reload. Thanks for your blog post! 💛

**Comment by henninghorn** [comment]
I'm looking forward to it, @centrual !

**Comment by centrual** [comment]
@valnub @henninghorn I updated the code for webpack-dev-server. Just hot module replacement not working, because of that it refreshes page on every change, but everything else is good.

I added phonegap support and i made lot of changes. Can you check? /play whoomp

Have a great day!

**Comment by nolimits4web** [comment]
@centrual maybe you can send PR to these pages http://framework7.io/vue/ (button with template link) and http://framework7.io/vue/templates.html with information about your template?

**Comment by centrual** [comment]
@nolimits4web Okay, i adding immediately 👍 Thank you! 😄

**Comment by ijry** [comment]
how to listen android's backbutton event and handle with this.$router.back();
@centrual

**Comment by hazzo** [comment]
Sorry but it's not simpler and cleaner to have a cordova folder inside the webpack template and set cordova there? I test it and it works. Just make a symlink of `www` folder with the `dist` folder and works great. The only downsize it is that you have to run commands inside the cordova folder.

@centrual I tested your template but I miss `npm run dev` because it does not trigger `cordova run|emulate` for hot reloading.

**Comment by hazzo** [comment]
@nolimits4web forked your webpack template and done slight modifications to add cordova wrapper. I modified slightly your README to explain some little "install" steps. I find your template very organised so I decided to add cordova to keep with the flow ;) 
I made this fork for my own purposes and I thought it could help someone.

[https://github.com/hazzo/framework7-vue-webpack-cordova-template](url)

Cheers!
